src/DIRAC/FrameworkSystem/Client/ComponentInstaller.py

```diff
@@ -586,10 +587,11 @@ def removeComponentOptionsFromCS(self, system, component, mySetup=None):
       else:
         isRenamed = True
 
-      result = self.monitoringClient.getInstallations({'UnInstallationTime': None},
-                                                      {'System': system,
-                                                       'Module': installation['Component']['Module']},
-                                                      {}, True)
+      result = self.monitoringClient.getInstallations(
+          {'UnInstallationTime': None},
+          {'System': system, 'DIRACModule': installation['Component']['Module']},
```

Comment:
@fstagni should it be `DIRACSystem` here ?

Reply:
Thanks, fixed in my open PR.